<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="prev_screen_button">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>221</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Vissza az előző elemzésekhez</string>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>60</y>
     <width>831</width>
     <height>471</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="top_pos_vbox_1"/>
  </widget>
  <widget class="QComboBox" name="pos_combo_box_1">
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>260</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pos_button_1">
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>300</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Megtekintés</string>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>530</y>
     <width>831</width>
     <height>471</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="top_pos_vbox_2"/>
  </widget>
  <widget class="QComboBox" name="pos_combo_box_2">
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>730</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pos_button_2">
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>770</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Megtekintés</string>
   </property>
  </widget>
  <widget class="QLabel" name="pos_label_1">
   <property name="geometry">
    <rect>
     <x>850</x>
     <y>370</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="pos_label_2">
   <property name="geometry">
    <rect>
     <x>850</x>
     <y>430</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_4">
   <property name="geometry">
    <rect>
     <x>970</x>
     <y>530</y>
     <width>951</width>
     <height>471</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="scatter_vbox"/>
  </widget>
  <widget class="QPushButton" name="corr_button">
   <property name="geometry">
    <rect>
     <x>1338</x>
     <y>180</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Számítás</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="count_textbox">
   <property name="geometry">
    <rect>
     <x>1208</x>
     <y>180</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="corr_text_label">
   <property name="geometry">
    <rect>
     <x>1198</x>
     <y>150</y>
     <width>500</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Korrelációk kiszámításához adj meg egy számot 10 és 1000 között.</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="corr_label">
   <property name="geometry">
    <rect>
     <x>1208</x>
     <y>231</y>
     <width>181</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Korreláció</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="coef_label">
   <property name="geometry">
    <rect>
     <x>1499</x>
     <y>230</y>
     <width>191</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Korrelációs koefficiens</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="p_value_label">
   <property name="geometry">
    <rect>
     <x>1398</x>
     <y>231</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>P-érték</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="pearson_p_value">
   <property name="geometry">
    <rect>
     <x>1398</x>
     <y>260</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="pearson_coef">
   <property name="geometry">
    <rect>
     <x>1499</x>
     <y>259</y>
     <width>191</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="pearson_label">
   <property name="geometry">
    <rect>
     <x>1208</x>
     <y>260</y>
     <width>181</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Pearson korreláció</string>
   </property>
  </widget>
  <widget class="QLabel" name="spearman_label">
   <property name="geometry">
    <rect>
     <x>1208</x>
     <y>290</y>
     <width>181</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Spearman rangkorreláció</string>
   </property>
  </widget>
  <widget class="QLabel" name="spearman_p_value">
   <property name="geometry">
    <rect>
     <x>1398</x>
     <y>290</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="spearman_coef">
   <property name="geometry">
    <rect>
     <x>1499</x>
     <y>289</y>
     <width>191</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="kendall_label">
   <property name="geometry">
    <rect>
     <x>1208</x>
     <y>321</y>
     <width>181</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Kendall rangkorreláció</string>
   </property>
  </widget>
  <widget class="QLabel" name="kendall_p_value">
   <property name="geometry">
    <rect>
     <x>1398</x>
     <y>321</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="kendall_coef">
   <property name="geometry">
    <rect>
     <x>1499</x>
     <y>320</y>
     <width>191</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="jaccard_value_label">
   <property name="geometry">
    <rect>
     <x>1388</x>
     <y>430</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="jaccard_label">
   <property name="geometry">
    <rect>
     <x>1208</x>
     <y>430</y>
     <width>181</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Jaccard-hasonlóság</string>
   </property>
  </widget>
  <widget class="QLabel" name="sim_value_label">
   <property name="geometry">
    <rect>
     <x>1388</x>
     <y>400</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Érték</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="similarity_label">
   <property name="geometry">
    <rect>
     <x>1208</x>
     <y>400</y>
     <width>181</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Hasonlóság</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="next_screen_button">
   <property name="geometry">
    <rect>
     <x>1750</x>
     <y>20</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>További elemzések</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>1201</x>
     <y>222</y>
     <width>488</width>
     <height>10</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>1201</x>
     <y>251</y>
     <width>488</width>
     <height>10</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_3">
   <property name="geometry">
    <rect>
     <x>1201</x>
     <y>280</y>
     <width>488</width>
     <height>10</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_4">
   <property name="geometry">
    <rect>
     <x>1201</x>
     <y>310</y>
     <width>488</width>
     <height>10</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_5">
   <property name="geometry">
    <rect>
     <x>1201</x>
     <y>340</y>
     <width>488</width>
     <height>10</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_6">
   <property name="geometry">
    <rect>
     <x>1196</x>
     <y>226</y>
     <width>10</width>
     <height>119</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="Line" name="line_7">
   <property name="geometry">
    <rect>
     <x>1389</x>
     <y>226</y>
     <width>10</width>
     <height>119</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="Line" name="line_8">
   <property name="geometry">
    <rect>
     <x>1489</x>
     <y>226</y>
     <width>10</width>
     <height>119</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="Line" name="line_9">
   <property name="geometry">
    <rect>
     <x>1685</x>
     <y>226</y>
     <width>10</width>
     <height>119</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QLabel" name="cosine_label">
   <property name="geometry">
    <rect>
     <x>1208</x>
     <y>460</y>
     <width>181</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Koszinusz-hasonlóság</string>
   </property>
  </widget>
  <widget class="QLabel" name="cosine_value_label">
   <property name="geometry">
    <rect>
     <x>1388</x>
     <y>460</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="Line" name="line_10">
   <property name="geometry">
    <rect>
     <x>1200</x>
     <y>390</y>
     <width>281</width>
     <height>10</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_11">
   <property name="geometry">
    <rect>
     <x>1200</x>
     <y>420</y>
     <width>281</width>
     <height>10</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_12">
   <property name="geometry">
    <rect>
     <x>1200</x>
     <y>450</y>
     <width>281</width>
     <height>10</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_13">
   <property name="geometry">
    <rect>
     <x>1200</x>
     <y>480</y>
     <width>281</width>
     <height>10</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_14">
   <property name="geometry">
    <rect>
     <x>1195</x>
     <y>394</y>
     <width>10</width>
     <height>91</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="Line" name="line_15">
   <property name="geometry">
    <rect>
     <x>1386</x>
     <y>394</y>
     <width>10</width>
     <height>91</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="Line" name="line_16">
   <property name="geometry">
    <rect>
     <x>1477</x>
     <y>394</y>
     <width>10</width>
     <height>91</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
